Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CartesianCKCAlgorithm.H

```diff
@@ -90,6 +92,20 @@ struct CartesianCKCAlgorithm {
         stencil_coefs_z[4] = gammaz*inv_dz;
     }
 
+    /**
+     * Compute the maximum timestep, for which the scheme remains stable
+     * (Courant-Friedrichs-Levy limit) */
+    static amrex::Real ComputeMaxDt ( amrex::Real const * const dx ) {
+#if (defined WARPX_DIM_XZ)
+            // - In Cartesian 2D geometry: determined by the minimum cell size in all direction
+            amrex::Real deltat = std::min( dx[0], dx[1] )/PhysConst::c;
+#else
+            // - In Cartesian 3D geometry: determined by the minimum cell size in all direction
+            amrex::Real deltat = std::min( dx[0], std::min( dx[1], dx[2] ) )/PhysConst::c;
+#endif
+        return deltat;
```

Comment:
If I had to nit-pick, but don't worry:
```suggestion
            amrex::Real const delta_t = std::min( dx[0], dx[1] )/PhysConst::c;
#else
            // - In Cartesian 3D geometry: determined by the minimum cell size in all direction
            amrex::Real const delta_t = std::min( dx[0], std::min( dx[1], dx[2] ) ) / PhysConst::c;
#endif
        return delta_t;
```